core/commands/log.py

```diff
@@ -250,5 +250,13 @@ def blame_file_atcommit(self):
         })
 
     def checkout_file_at_commit(self):
-        self.checkout_ref(self._commit_hash, fpath=self._file_path)
+        commit_hash = self._commit_hash
+        file_path = self._file_path
+
+        if commit_hash and file_path and self.savvy_settings.get("log_follow_rename"):
```

Comment:
At this point all three predicates should be `True` all the time if I'm reading this correctly.  

There is also https://github.com/timbrel/GitSavvy/blob/8afe2de4e8c9cb5407bd63fb649c9b604cbd5faa/core/commands/log_graph.py#L2702-L2704 which would need the same patch I think.

Reply:
Do you mean the three expressions in the `if` statement, `if commit_hash and file_path and self.savvy_settings.get("log_follow_rename"):`?

Type checker says `file_path` is `Unknown | None` in line 254 above, so I figured, and `self.filename_at_commit` takes `(str, str)`, so I figured I'd just make sure both were truthy

As for `self.savvy_settings.get("log_follow_rename")`, that's a user setting that's not guaranteed to be truthy in general, right?

---

Agreed on making same patch in `log_graph`.py` module, I'll do that

Reply:
> As for self.savvy_settings.get("log_follow_rename"), that's a user setting that's not guaranteed to be truthy in general, right?

Ah right, looks like `log_follow_rename` can be `False` still, but `self._commit_hash` is always truthy here and `self._file_path` too otherwise the menu item would not appear (see `def update_actions(self)`).